pkg/inttest/http.go

```diff
@@ -171,6 +171,41 @@ func (hc *HTTPClient) PostJSON(t *testing.T, path string, requestBody io.Reader,
 	require.NoError(t, err, errMsg+": failed to unmarshal response body")
 }
 
+func (hc *HTTPClient) SignIn(t *testing.T, email, password string) (*http.Cookie, *http.Cookie) {
+	t.Helper()
+
+	req := hc.NewRequest(t, http.MethodPost, "/tokens", nil, WithHeader("Content-Type", "application/json"), WithBasicAuth(email, password))
+	res := hc.do(t, req)
+	require.Zero(t, res.ContentLength)
+
+	errMsg := httpClientErrMessage(http.MethodPost, "/tokens")
+	defer func() {
+		require.NoError(t, res.Body.Close(), errMsg+": failed to close HTTP response body")
+	}()
+	require.Equal(t, http.StatusCreated, res.StatusCode, errMsg+": HTTP status mismatch")
+
+	body, err := io.ReadAll(res.Body)
+	require.Len(t, body, 0)
+
+	accessToken, err := findCookie(res, "accessToken")
+	require.NoError(t, err)
+
+	refreshToken, err := findCookie(res, "refreshToken")
+	require.NoError(t, err)
+
+	return accessToken, refreshToken
+}
+
+func findCookie(res *http.Response, name string) (*http.Cookie, error) {
+	cookies := res.Cookies()
```

Comment:
Such vars do not add value (var is named like the method) and just increase the scope of the var